rust/main/helm/hyperlane-agent/templates/relayer-statefulset.yaml

```diff
@@ -17,7 +17,9 @@ spec:
     metadata:
       annotations:
         checksum/configmap: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/external-secret: {{ include (print $.Template.BasePath "/external-secret.yaml") . | sha256sum }}
```

Comment:
drive-bys to make sure pods get updated if doing a release and there's a checksum change of the external secrets